src/app/wizard/service/wizard.ts

```diff
@@ -48,18 +48,11 @@ export class WizardService {
       }
     }
 
-    handleOSChange(os: OperatingSystem): void {
+    handleOSChange(_os: OperatingSystem): void {
       if (this._parent._clusterService.provider !== NodeProvider.VSPHERE) {
         this._hideStep(StepRegistry.MachineNetwork);
         return;
       }
-
-      if (os !== OperatingSystem.ContainerLinux) {
-        this._hideStep(StepRegistry.MachineNetwork);
-        return;
-      }
-
-      this._showStep(StepRegistry.MachineNetwork);
```

Comment:
If I understand correctly we no longer need the machine network step at all or is there any provider/os combo that still requires it?

Reply:
I don't even see this in UI (original code, before this changes) :D

Reply:
<img width="1568" alt="Zrzut ekranu 2021-02-4 o 09 48 50" src="https://user-images.githubusercontent.com/2823399/106867698-34cbce00-66ce-11eb-97fe-4eed9cbbed24.png">

It's visible only for vSphere and Container Linux at the moment (@kgroschoff correct me if I'm wrong).

Reply:
Exactly. It was a dynamically added step based on certain specific conditions and it easy only there for Container Linux.

Reply:
Basically, this part will be a dead code right now :slightly_smiling_face: We will not be using this dynamic step functionality anymore.

Reply:
@maciaszczykm how should "StepRegistry.MachineNetwork" wizard step look like?

Reply:
@kron4eg try it on dev. Select vsphere as the provider and then container linux as the os. The next step will be dynamically added.

Reply:
If this (vSphere + Container Linux) was the only combination where it could be accessed then it should be completely removed I think. @kgroschoff, can you confirm this?

Reply:
Yes, agree. It was only implemented for vSphere + Container Linux.